extras/02_kafka.sh

```diff
@@ -39,7 +39,7 @@ oc wait --for condition=ready pod -l name=strimzi-cluster-operator -n ${KAFKA_NA
 
 # Modify Kafka cluster & Deploy
 sed -i "s/my-cluster/${KAFKA_CLUSTERNAME}/" metrics/examples/kafka/kafka-metrics.yaml
-sed -i "s/100Gi/${KAFKA_PVC_SIZE}Gi/" metrics/examples/kafka/kafka-metrics.yaml
+sed -i "s/1Gi/${KAFKA_PVC_SIZE}Gi/" metrics/examples/kafka/kafka-metrics.yaml
```

Comment:
that should still be 100Gi for this sed

Reply:
it is, but applied by a variable set at the head of the file:

```
KAFKA_PVC_SIZE=${KAFKA_PVC_SIZE:-100}
￼```